src/visbook/visbook.api.ts

```diff
@@ -7,9 +8,15 @@ Visbook API docs: https://ws.visbook.com/8/docs/index.html
 export class VisbookApi {
   async getAccommodationAvailability(
     cabinVisbookId: number,
-    checkIn: string,
-    checkOut: string,
+    bookingDates: BookingDates,
   ): Promise<AccommodationAvailability> {
+    const formatAsYYYYMMDD = (date: Date) => {
+      // Target format is yyyy-mm-dd
+      return date.toISOString().split('T')[0];
+    };
```

Comment:
Could make this function a file constant. 

This here is an anonymous function which means if an error occurs you won't get a good stack trace. Using a named function would be better IMO. (nit pick).

Reply:
Not sure if that is what you meant, but I moved it to newly created visbook utils and made it a regular named function.

Reply:
Good call!